docs/python.rst

```diff
@@ -31,7 +31,7 @@ To calculate a 1D type 1 transform, from nonuniform to uniform points, we import
     f = finufftpy.nufft1d1(x, c, N)
 
 The input here is a set of complex strengths ``c``, which are used to approximate (1) in :ref:`math`.
-That approximation is stored in ``f``, which is indexed from ``-N/2`` up to ``N/2-1`` (since ``N`` is even; if odd it would be ``-(N-1)/2`` up to ``(N-1)/2``).
+That approximation is stored in ``f``, which is indexed from ``-N // 2`` up to ``N // 2 - 1`` (since ``N`` is even; if odd it would be ``-(N - 1) // 2`` up to ``(N - 1) // 2``).
```

Comment:
I'm a little confused why you changed this back: N is even, so / and // are equiv. I corrected the odd case and split it out because it's confusing and we don't want to rely on the reader understanding the language behavior. Case in point: in py3,  3//2=1 but -3//2=-2. In C, -3/2 = -1. Fun, eh?

Reply:
You're right, but there's a slight difference. In Python `4 // 2` is equal to `2` (an integer) while `4 / 2` is `2.0` (a float). This comes up (for example), if you pass it to `np.zeros`, which expects integers. At any rate, maybe it's just confusing here.